### DISCUSSION TITLE
Empty world when using AnvilLoader

### DISCUSSION THREAD

**Discussion by DearFox** [Minestom/Minestom#980] [Help] [answered]
After experimenting with the world generator, I decided to load ready-made chunks using setChunkLoader

`instanceContainer.setChunkLoader(new AnvilLoader(Path.of("lobby")));`

found ready-made regions of the world:
![image](https://user-images.githubusercontent.com/59219907/165149036-9da9dfe8-7915-489c-995b-11e6409e0a0f.png)
_(this is the lobby from Minestom/Arena)_

But instead of a lobby, I got a void
![image](https://user-images.githubusercontent.com/59219907/165149525-e7269db7-5e82-4e06-a355-eb30aa66f48e.png)
I do not really understand what I did wrong and maybe I forgot about something?

**Comment by MrlnHi** [comment, marked as answer]
don't put the world directory inside of resources but at the jar path (or the run path if you're running inside IDE)

**Reply by DearFox** [reply]
Yes, thanks, I didn’t notice how it is implemented in Minestom / Arena
![image](https://user-images.githubusercontent.com/59219907/165152324-06f5f394-7464-4d9a-9543-3dfa57980623.png)
This solved the problem.

**Reply by TheMode** [reply]
The arena server copy the folder from resource https://github.com/Minestom/Arena/blob/ba8ca128d97bcb3a4dc79224a3aa3b4add80b9ca/src/main/java/net/minestom/arena/Main.java#L37 on startup.

It is to allow automatic update directly from the jar.
